site/_posts/2019-10-08-Velero-v1-1-on-vSphere.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Velero v1.1 backing up and restoring apps on vSphere
+image: /img/posts/vsphere-logo.jpg
+excerpt: A How-To guide to run Velero on vSphere.
+author_name: Cormac Hogan
+author_avatar: /img/contributors/cormac-pic.png
+categories: ['kubernetes']
+# Tag should match author to drive author pages
+tags: ['Velero', 'Cormac Hogan', 'how-to']
+---
+Velero version 1.1 provides support to backup Kubernetes applications deployed on vSphere. This post will provide detailed information on how to install and configure Velero to backup and restore a stateless application (`nginx`) that is running in Kubernetes on vSphere. At this time there is no vSphere plugin for snapshotting stateful applications on vSphere during a Velero backup. In this case, we rely on a third party program called `restic`. However this post does not include an example of how to backup a stateful application. That is available in another tutorial which can be found [here](../Velero-v1-1-Stateful-Backup-vSphere).
+
+## Overview of steps
+
+* Download and extract Velero v1.1
+* Deploy and Configure a Minio Object store
+* Install Velero using the `velero install` command, ensuring that both `restic` support and a Minio `publicUrl` are included
+* Run a test backup/restore of a stateless application that has been deployed on upstream Kubernetes
+
+## What this post does not show
+
+* How to install Velero v1.1 with PKS, Pivotal Container Service, on vSphere
```

Comment:
Just a tiny nitpick, this line is unneeded.

Reply:
Done